boards/arm64/xenvm/xenvm-gicv2-qemu_defconfig

```diff
@@ -0,0 +1,31 @@
+CONFIG_SOC_XENVM=y
```

Comment:
Do we actually need whole new defconfig for just 1 changed option? I think it is better to left default GICv3 for xenvm_qemu and, if someone will need GICv2 in qemu, just change this value on own setup.

Reply:
There is more then 1 option.
There is different reg value in dts. I wanted to do this switcher by option but I can't use different dts for this. Some reason dts processed before any of CMakeList.txt I can change (in boards/arm64/xenvm/ and application directories)